### DISCUSSION TITLE
【憂鬱症vs憂鬱情緒】

### DISCUSSION THREAD

**Discussion by Chienyuuu6** [1101-GS4719-C/Python-Community-GS4719-C#1314] [General]
#MentalHealthAwarenessMonth

你們知道在美國五月是Mental Health Awareness Month嗎？
近期來跟大家分享有關憂鬱還有自我照顧的方法！

我以前還是球員的時候，自己的情緒總是跟著球場的表現和球隊狀況一起波動～直到上了大學後才真正感受到什麼是來自生活上的「狀況不好」、「低潮期」，那種徬徨無助的感覺⋯
（以前面對球場的低潮期雖然也會很憂鬱，但是那個對籃球純粹的夢想和熱情真的是支撐著我一直往前的動力🥹）

在開始介紹自我照顧的方法之前，先來了解憂鬱情緒和憂鬱症的不同～

憂鬱症 vs 憂鬱情緒

   
1. 程度差異大
- 持續時間
（2星期以上/較短時間）
- 影響生活的程度
（e.g.沒辦法去上學/可以但是超不想上學）

2. 成因不同
- 憂鬱情緒通常有明確事件

3. 自我控制感
- 做任何事情都辦法法讓自己好一點

憂鬱症是什麼？
心理疾病 vs 生理疾病
憂鬱症的成因有心理因素，也有生理因素。但究竟是哪個層面為主要的誘發因素到現在還沒有明確的說法！
🧠生理
1. 神經傳導物質
（思考窄化，影響解決事情的能力等）
2. 基因遺傳
🫀心理
1. 心理因素
（壓力事件、人格特質）
2. 社會因素
（人際連結）
💡單一因素不一定會造成憂鬱症！

憂鬱症可能有的感受

- 失去快樂的能力
（做了好多事情但都不會感到開心快樂）
- 難以有正向期待和希望感
- 失去自我價值
- 無能為力
- 吞噬感
（被黑色情緒淹沒）

🪫憂鬱症就像電池壞掉的人

**Comment by TW-Maverick** [comment]
感謝你的分享，其實我有時候會突然做什麼事都提不起勁，只想耍廢，什麼都不幹的時候!
我不太確定這是不是憂鬱情緒，但過了兩三天，等遊戲玩膩了後，又可以開始讀書了XD
很常發生在開學後幾個禮拜或期中考完到期末考之間

**Comment by tower20021202** [comment]
如你所說憂鬱症就像電池壞掉的人的話，那我時常感到憂鬱情緒的話應該可說是電池快沒電了吧，只要過段時間等電充飽，隔天依然是個充滿希望的日子

**Comment by Evening218** [comment]
之前看過的一本小說曾經寫過憂鬱症,他說憂鬱症就是長期對生活對任何事物失去動力,甚至失去了無論是正面還是負面的情緒波動,我覺得憂鬱症就像電池壞掉的人是個很好的比喻,失去了蓄電能力,也無法透過各種喜歡的事物給自己充電
